tools/c7n_mailer/c7n_mailer/aws/aws_smtp_delivery.py

```diff
@@ -0,0 +1,17 @@
+from c7n_mailer.smtp_delivery import SmtpDelivery
+import c7n_mailer.utils as utils
+
+
+class AwsSmtpDelivery(SmtpDelivery):
```

Comment:
please remove the provider specific transports, the only difference here is the kms decrypt, which is intentionally intended to fail safe to the original value.

Reply:
`kms_decrypt` fallback code works only if AWS is the only provider allowed to encrypt secrets.

I though we will eventually add KeyVault encryption for Azure and something for gcp..
When we do so, fallback logic in `kms_decrypt` is no longer sufficient.